cpp/include/cudf_test/timestamp_utilities.cuh

```diff
@@ -55,7 +55,6 @@ inline cudf::test::fixed_width_column_wrapper<T, int64_t> generate_timestamps(in
   auto lhs = start.time_since_epoch().count();
   auto rhs = stop.time_since_epoch().count();
 
-  // When C++17, auto [min, max] = std::minmax(lhs, rhs)
   auto min   = std::min(lhs, rhs);
   auto max   = std::max(lhs, rhs);
   auto range = max - min;
```

Comment:
Suggest adding `const` qualifier for the variables.

Reply:
Done.